backend/stakepoold/rpc/rpcserver/server.go

```diff
@@ -33,8 +33,8 @@ const (
 	// collection cycle to also trigger a timeout but the current allocation
 	// pattern of stakepoold is not known to cause such conditions at this time.
 	GRPCCommandTimeout = time.Millisecond * 100
-	semverString       = "5.0.0"
-	semverMajor        = 5
+	semverString       = "7.0.0"
```

Comment:
Did you mean to increment this by two?

Reply:
I guess I was looking at a cached version. It was already changed before I wrote this. Srry.

Reply:
No worries, I think you might have been reading it as I was making the change :)